Azure/azure-cli
# [AppService] Fixes #15604, #15605: Add Dotnet5 support

**Description**  


Fixes #15604 #15605 

- Add .NET 5 support for AppService webapps. Runtime `dotnet|5.0` can be used with `az webapp create` and `az webapp up` 

**Testing Guide**  
- `az webapp list-runtimes (--linux)` should show `dotnet|5.0` as an available runtime
- `az webapp up` run in a .NET5 app should work and create a `dotnet|5.0` webapp
- `az webapp create --runtime "dotnet|5.0"` should create a `dotnet|5.0` webapp

Also included bugfixes:
- issue #15701: running `az webapp create --runtime "node|12-lts"` consecutively with the same webapp should work
- Also fixes issue where running `az webapp up` on an existing webapp with different runtime doesn't update the webapp's runtime


---

This checklist is used to make sure that common guidelines for a pull request are followed.
